README.md

```diff
@@ -105,8 +105,6 @@ You can then invoke `swiftgen` directly in your Script Build Phase (as it will b
 swiftgen … 
 ```
 
-_Note: SwiftGen needs Xcode 8.3 to build, so installing via Homebrew requires you to have Xcode 8.3 installed (which in turn requires macOS 10.12). If you use an earlier version of macOS, you'll have to use one of the other installation methods instead._
-
```

Comment:
Removed this as:
 - the Xcode minimum version isn't the right one anymore (I think we're 11.4+ since recently now)
 - it doesn't really matter anyway with homebrew install, as homebrew pre-compiles SwiftGen on their Jenkins and deliver it as a _"bottle"_ when installing on the user's machine. (It's only if a bottle is not available in homebrew for the user's macOS older version that SwiftGen would be built on the user's machine and require an up-to-date Xcode)